<launch>    
        <node ns="skid4wd" pkg="stereo_image_proc" type="stereo_image_proc" name="stereo_image_proc" output="screen">
            <remap from="/skid4wd/stereo_camera/left/image_raw" to="left/image_raw"/>
            <remap from="/skid4wd/stereo_camera/right/image_raw" to="right/image_raw"/>
            <remap from="/skid4wd/stereo_camera/left/camera_info" to="left/camera_info"/>
            <remap from="/skid4wd/stereo_camera/right/camera_info" to="right/camera_info"/>
            <param name="approximate_sync" value="true"/>
            <param name="queue_size" value="10"/>
        </node>

        <node pkg="image_view" type="stereo_view" name="stereo_view">
            <param name="approximate_sync" value="true"/>
            <param name="queue_size" value="100"/>
            <remap from="cameras_pan_tilt" to="stereo"/>
            <remap from="image_rect" to="image"/>
        </node>

        <node pkg="rqt_reconfigure" type="rqt_reconfigure" name="rqt_reconfigure"/>
</launch>
